btr-web/btr-main-app/cypress/e2e/pages/ownerChange.cy.ts

```diff
@@ -77,20 +77,6 @@ describe('pages -> Beneficial Owner Change', () => {
     cy.get('[data-cy=add-new-btn]').should('not.have.attr', 'disabled')
   })
 
-  it('has expected done button functionality with add individual', () => {
```

Comment:
how come you removed this test? Is it covered elsewhere?

Reply:
In summaryTable.cy.ts and reviewConfirm.cy.ts, we fill the full form and add some individuals. So the happy path for DONE button has been covered.